README.md

```diff
@@ -1,23 +1,35 @@
-# Transformer Template
-A template for creating transformers for multiple environments.
+# Transformer OpenDroneMap
 
-## Quick Start
-To use this template:
-1. Clone this template into a new repository
-2. Fill out the `configuration.py` file with your information. Feel free to add additional variables.
-3. Add your code to the `transformer.py` file, filling in the *add_parameters*, *check_continue*, and *perform_process* functions.
-4. Run the `generate-docker.py` script to generate your Dockerfile for building images
-5. Build the Docker image for your transformer, being sure to specify the desired source image
+Uses OpenDroneMap to process drone captured RGB data.
 
-For your transformer to be accepted, be sure to have test cases and continuous integration setup.
-Please be sure to read about how to contribute in the documents held in our [main repository](https://github.com/AgPipeline/Organization-info).
+### Sample Docker Command line
+Below is a sample command line that shows how the OpenDroneMap Docker image could be run.
+An explanation of the command line options used follows.
+Be sure to read up on the [docker run](https://docs.docker.com/engine/reference/run/) command line for more information.
 
-## Extending the Template
-There are situations where this template won't be sufficient as a transformer for an environment.
-In these cases it's recommended that instead of forking this repo and making modifications, a new template repo is created with the expectation that the processing code will be a submodule to it.
-Scripts and/or instructions can then be provided on cloning this repo, specifying the submodule, and how to create a working transformer for the environment.
+```sh
```

Comment:
FYI you can have markdown make code pretty if you tell it what language ... I made this change:

![image](https://user-images.githubusercontent.com/464871/70359878-38e62c80-183a-11ea-83a7-edccdd287fe3.png)

Reply:
Great!